src/physics/base/PhysicsParams.cc

```diff
@@ -233,7 +232,7 @@ void PhysicsParams::build_ids(const ParticleParams& particles,
         pgroup.models    = model_groups.insert_back(temp_model_groups.begin(),
                                                  temp_model_groups.end());
         // NOTE: data tables will be assigned later
-        CELER_ASSERT(pgroup);
+        CELER_ASSERT(process_to_models.empty() || pgroup);
```

Comment:
Oh you know what, we might not have talked about this one. This would be ok for particles that don't have any processes defined, i.e. any logged on line 176.

Reply:
[for the record] We discussed in Monday's meeting: this is essentially because of an inconsistency between the listed particles and the processes written to the file.